packages/@lwc/ssr-runtime/src/reflection.ts

```diff
@@ -127,9 +130,10 @@ export function reflectAttrToProp(
     const reflectedPropName = attrsToProps[attrName as keyof typeof attrsToProps];
     // If it is a reflected property and it was not overridden by the instance
     if (reflectedPropName && !hasOwnProperty.call(instance, reflectedPropName)) {
-        const currentValue = (instance as any)[reflectedPropName];
+        const currentValue = instance[reflectedPropName];
         if (currentValue !== attrValue) {
-            (instance as any)[reflectedPropName] = attrValue;
+            // Type assertion because we're providing a string, but some props accept boolean/number
+            (instance[reflectedPropName] as string | null) = attrValue;
```

Comment:
Looks like the type of `reflectedPropName` does not include all the aria property names because `attrsToProps` is missing the aria mapping?